scripts/west_commands/create_board/templates/nrf52/board.cmake.jinja2

```diff
@@ -1,10 +1,13 @@
-board_runner_args(jlink "--device={{ soc | replace("nrf", "nRF") }}_xx{{ variant[2:] | upper }}" "--speed=4000")
+if(CONFIG_SOC_NRF52840_QIAA)
```

Comment:
this should only be present if the soc is nrf52840, not as in "if(CONFIG..." in the output file but just the one line if soc == nrf52840. Also this depends on if they have QSPI set up or even wired in...

Reply:
Regarding having QSPI set up or wired in, that's just a starting point for the board and they'll have to refine it, so I we are just being complete here for the customer using it